### PR TITLE
Docs - add info for using gp_log_suboverflow_statements GUC

### PR DESCRIPTION
**Pull request by dyozie**

### CODE REVIEW

**Review by VMWareJennyBeth — APPROVED** [review]
LGTM

File: gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md [hunk 1]
```diff
@@ -972,6 +972,14 @@ The information is written to the server log.
 |-----------|-------|-------------------|
 |Boolean|false|master, session, reload|
 
+## <a id="gp_log_suboverflow_statements"></a>gp\_log\_suboverflowed\_statements
+
+Controls whether Greenplum logs statements that cause subtransaction overflow to the `gp_toolkit.gp_log_system` table. See [Checking for and Terminating Overflowed Backends](/oss/admin_guide/managing/monitor.html).
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:977]: Lets take out: ~to the `gp_toolkit.gp_log_system` table~

**Comment by soumyadeep2007** [comment]
Also, it might be better to put this into master first as the exact same thing has been merged into master recently (unlike the other subtx overflow identification components)

**Comment by dyozie** [comment]
> Also, it might be better to put this into master first as the exact same thing has been merged into master recently (unlike the other subtx overflow identification components)

Right now the docs for the feature are only in 6X_STABLE. However, I've discussed this with @VMWareJennyBeth and she's aware that the docs in this PR (as well as her original PR) will need to be adapted as appropriate for inclusion with v7.x in master.

**Comment by VMWareJennyBeth** [comment]
@soumyadeep2007 @dyozie 
Yes, affirming that I will be looking at the 6x suboverflowed stuff and figuring out what to keep and what to toss for 7x. Once I can get out from under Cluster Recovery Beta. Thank you.

**Pull request merged**